builder/repo_control.go

```diff
@@ -351,7 +354,13 @@ func (b *Builder) ListRepos() error {
 			pURL.Scheme = "file"
 		}
 
-		fmt.Printf("%s\t%s\n", name, pURL.String())
+		priority := s.Key("priority").Value()
+		if priority == "" {
+			// When the priority field is omitted, the DNF default is 99.
+			priority = "99"
```

Comment:
Should we keep dnf default and our default priority same ?

Reply:
From the perspective of a downstream mix, I prefer the current default behavior where the priority defaults to 1. This allows repos added by the user to override the upstream clear repo by default.